tgui/packages/tgui/interfaces/PowernetDebugger.js

```diff
@@ -0,0 +1,371 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, Section, Table, Box, LabeledList, Collapsible, ProgressBar, Tabs, Icon } from '../components';
+import { Window } from '../layouts';
+
+export const PowernetDebugger = (props, context) => {
+  return (
+    <Window resizable width={1000} height={750}>
+      <Window.Content scrollable>
+        <DebuggerNavigation />
+        <DebuggerContent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const DebuggerNavigation = (properties, context) => {
+  const { data, act } = useBackend(context);
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 'powernets');
+  const { selected_nets } = data;
+  return (
+    <Tabs>
+      <Tabs.Tab
+        selected={'powernets' === tabIndex}
+        onClick={() => {
+          setTabIndex('powernets');
+          act('set_page', { 'page': 1 });
+        }}
+      >
+        <Icon name="list" />
```

Comment:
```suggestion
      <Tabs.Tab
        icon="list"
        selected={'powernets' === tabIndex}
        onClick={() => {
          setTabIndex('powernets');
          act('set_page', { 'page': 1 });
        }}
      >
```

You can put icon directly to tab